apps/overlay/src/views/lowerthirds/CustomCarousel.o.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react"
```

Comment:
Remove `React` import, this will import the whole library of react and can cause performance issues, we need the overlay side as fast as possible